### PR TITLE
[ KTL-130 ] Add mention of JB on first screen

### PR DESCRIPTION
**Pull request by JuliaVolkova**

### CODE REVIEW

File: static/js/page/index/kotlin-intro-section.scss [hunk 1]
```diff
@@ -3,29 +3,62 @@
...
-}
+  letter-spacing: -0.02em;
 
-.kotlin-intro-section__icon {
-  font-size: 0;
-  line-height: 0;
-  text-indent: -9999px;
-  display: inline-block;
-}
+  /* white */
```
Comment (zoobestik) [inline static/js/page/index/kotlin-intro-section.scss:14]: )))

File: mysettings.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 BLOG_URL = 'https://blog.jetbrains.com/kotlin/'
 FORUM_URL = 'https://discuss.kotlinlang.org/'
 BUGTRACKER_URL = 'https://youtrack.jetbrains.com/issues/KT'
+KOTLIN_CONTRIBUTORS = 'https://github.com/JetBrains/kotlin/graphs/contributors'
```
Comment (zoobestik) [inline mysettings.py:21]: Commonly it's used for links used more than one time.

File: templates/inc/pages/index/intro.html [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
     <div class="g-layout kotlin-intro-section">
         <p class="kotlin-intro-section__intro">
-            A modern programming language<br>
-            that makes developers happier.<br>
-            Open source forever <a target="_blank" href="{{ config.GITHUB_URL }}" class="kotlin-intro-section__icon">Github</a>
+            A modern programming <br>
+            language that makes <br>
+            developers happier.<br>
+
+                <span class="kotlin-intro-section__contributors">
```
Comment (zoobestik) [inline templates/inc/pages/index/intro.html:8]: I'm not fan of semanticб but it's better looks `div.kotlin-intro-section__intro > p + p.kotlin-intro-section__contributors`.

File: static/js/page/index/kotlin-intro-section.scss [hunk 4]
```diff
@@ -3,29 +3,62 @@
...
+    display: flex;
+    align-items: center;
+    color: #fff;
+    margin-top: 22px;
+    letter-spacing: 0.005em;
+    -webkit-font-smoothing: antialiased;
+
+
+    .kotlin-intro-section__logo-link {
+      display: flex;
```
Comment (zoobestik) [inline static/js/page/index/kotlin-intro-section.scss:34]: What for?

File: static/js/page/index/kotlin-intro-section.scss [hunk 5]
```diff
@@ -3,29 +3,62 @@
...
+      margin-right: 24px;
+      width: 32px;
```
Comment (zoobestik) [inline static/js/page/index/kotlin-intro-section.scss:36]: Why is it twice in `.kotlin-intro-section__logo` and here?

File: static/js/page/index/kotlin-intro-section.scss [hunk 6]
```diff
@@ -3,29 +3,62 @@
...
+    text-decoration: none;
+    transition: 0.3s ease-out;
+  }
+
+  .kotlin-intro-section__logo {
+    display: inline-block;
+    width: 32px;
+    height: 32px;
+    background-size: contain;
+    padding-right: 24px;
```
Comment (zoobestik) [inline static/js/page/index/kotlin-intro-section.scss:60]: Can you explain why the padding exists?
I think, it's already have as margin in `.kotlin-intro-section__contributors` as **22px**.

File: static/js/page/index/kotlin-intro-section.scss [hunk 7]
```diff
@@ -3,29 +3,62 @@
...
+    color: #fff;
+    border-bottom: 1px solid #fff;
+    text-decoration: none;
+    transition: 0.3s ease-out;
+  }
+
+  .kotlin-intro-section__logo {
+    display: inline-block;
+    width: 32px;
+    height: 32px;
```
Comment (zoobestik) [inline static/js/page/index/kotlin-intro-section.scss:58]: In markup?

File: static/js/page/index/kotlin-intro-section.scss (hunk 1 above)
Comment (JuliaVolkova) [reply static/js/page/index/kotlin-intro-section.scss:14]: )))
Removed it :)

File: static/js/page/index/kotlin-intro-section.scss (hunk 4 above)
Comment (JuliaVolkova) [reply static/js/page/index/kotlin-intro-section.scss:34]: Fixed.

File: static/js/page/index/kotlin-intro-section.scss (hunk 7 above)
Comment (JuliaVolkova) [reply static/js/page/index/kotlin-intro-section.scss:58]: Add it in markup, removed from here.

File: static/js/page/index/kotlin-intro-section.scss (hunk 6 above)
Comment (JuliaVolkova) [reply static/js/page/index/kotlin-intro-section.scss:60]: Yes, it's redundant, removed it.

File: static/js/page/index/kotlin-intro-section.scss (hunk 5 above)
Comment (JuliaVolkova) [reply static/js/page/index/kotlin-intro-section.scss:36]: Removed it from `.kotlin-intro-section__logo`, added it in markup instead, and from here too, it's redundant.

File: templates/inc/pages/index/intro.html (hunk 3 above)
Comment (JuliaVolkova) [reply templates/inc/pages/index/intro.html:8]: Fixed.

File: mysettings.py (hunk 2 above)
Comment (JuliaVolkova) [reply mysettings.py:21]: Add link in markup.

**Review by zoobestik — APPROVED** [review]

**Pull request merged**